site/src/pages/[shortname]/blocks/[blockSlug].page.tsx

```diff
@@ -101,6 +102,18 @@ const parseQueryParams = (params: BlockPageQueryParams) => {
   return { shortname, blockSlug };
 };
 
+// Show `github.com/org/repo` instead of full URL with protocol, commit hash and path
+const generateRepositoryUrl = (repository: string): string => {
+  const repositoryUrlObject = new URL(repository);
+  const displayUrl = `${repositoryUrlObject.hostname}${repositoryUrlObject.pathname}`;
```

Comment:
I understand the point about port, which can be utilized via `host` instead of `hostname`. Since we're trying to not display all of the URL information here, as we're hiding the commit and branch for GitHub URLs, doesn't it make sense to hide the extra info, eg, query params, for other URLs as well?

This URL is just used for display data, not as the `href`.

Reply:
Oh right, makes sense! 👍